app/components/concept-admin/questions-page/question-form.ts

```diff
@@ -39,6 +43,30 @@ export default class QuestionFormComponent extends Component<Signature> {
   async handleOptionDeleted(optionIndex: number) {
     this.args.question.options = this.args.question.options.filter((_, currentOptionIndex) => currentOptionIndex !== optionIndex);
   }
+
+  @action
+  moveOptionDown(optionIndex: number) {
+    const options = this.options.slice();
+
+    if (optionIndex < options.length - 1) {
+      const temp = options[optionIndex + 1];
+      options[optionIndex + 1] = options[optionIndex] as Option;
+      options[optionIndex] = temp as Option;
```

Comment:
This can be simpler (I think?)

```javascript
[options[optionIndex+1], options[optionIndex]] = [options[optionIndex], options[optionIndex+1]]
```

Based on https://stackoverflow.com/a/872317/3189203